phpcs.xml

```diff
@@ -2,24 +2,23 @@
 <ruleset name="FreshRSS">
 	<arg name="extensions" value="php,phtml"/>
 	<arg name="tab-width" value="4"/>
-	<exclude-pattern>./.git/</exclude-pattern>
-	<exclude-pattern>./data/config.php</exclude-pattern>
-	<exclude-pattern>./data/update.php</exclude-pattern>
-	<exclude-pattern>./data/users/*/config.php</exclude-pattern>
-	<exclude-pattern>./extensions/</exclude-pattern>
-	<exclude-pattern>./lib/http-conditional.php</exclude-pattern>
-	<exclude-pattern>./lib/marienfressinaud/</exclude-pattern>
-	<exclude-pattern>./lib/phpgt/</exclude-pattern>
-	<exclude-pattern>./lib/phpmailer/</exclude-pattern>
-	<exclude-pattern>./lib/simplepie/</exclude-pattern>
-	<exclude-pattern>./node_modules/</exclude-pattern>
-	<exclude-pattern>./p/scripts/vendor/</exclude-pattern>
-	<exclude-pattern>./vendor/</exclude-pattern>
+	<exclude-pattern>/\.git/*</exclude-pattern>
+	<exclude-pattern>/data/config.php</exclude-pattern>
+	<exclude-pattern>/data/update.php</exclude-pattern>
+	<exclude-pattern>/data/users/*/config.php</exclude-pattern>
+	<exclude-pattern>/(?-i:extensions)/*</exclude-pattern><!-- Case insensitive -->
```

Comment:
This is because the same rules are also used for our `Extensions` repo, and without being case-sensitive, that rule would exclude the full repo due to its name, since it is apparently not possible to match using an absolute path from the root of the repo